pkg/controllers/management/eks/eks_cluster_handler.go

```diff
@@ -264,13 +264,6 @@ func (e *eksOperatorController) onClusterChange(key string, cluster *mgmtv3.Clus
 			}
 		}
 
-		// check for unauthorized error
-		if failureMessage != "" {
-			if strings.Contains(failureMessage, "403") {
-				return e.setFalse(cluster, apimgmtv3.ClusterConditionUpdated, "cannot access EKS, check cloud credential")
-			}
-		}
-
```

Comment:
Why'd we remove this error for create ? The below addition is in the upgrading block.

Reply:
eks-operator will now be in a failed updating state if this or any other error is encountered, but I do need to convert this error regardless of state, so I moved that to the top